MiniAn/minian_main.py

```diff
@@ -517,7 +535,7 @@ def save_minian_to_doric(
     res = Y - AC # residual images
 
     duration = Y.shape[0]
-    time_ = np.arange(0, duration/fr, 1/fr, dtype="float64")
+    time_ = np.arange(0, duration/freq, 1/freq, dtype="float64")
```

Comment:
@doriclenses  Do I also load time from file here too?
I could add `time` as parameter and load it just before calling save_minian_to_doric.
Or I could load it on the fly there

Reply:
yepp